website/versioned_docs/version-0.67/integration-with-android-fragment.md

```diff
@@ -17,12 +19,54 @@ You will need to implement the ReactApplication interface in your main Applicati
 
 Ensure your main Application Java class implements ReactApplication:
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin
+class MyReactApplication: Application(), ReactApplication {...}
+```
+
+</TabItem>
+<TabItem value="java">
+
 ```java
 public class MyReactApplication extends Application implements ReactApplication {...}
 ```
 
+</TabItem>
+</Tabs>
+
 Override the required methods `getUseDeveloperSupport`, `getPackages` and `getReactNativeHost`:
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin
+class MyReactApplication: Application(), ReactApplication {
+
+    override fun onCreate() {
+        super.onCreate()
+        SoLoader.init(this, false)
+    }
+
+    private val reactNativeHost = object : ReactNativeHost(this) {
+
+        override fun getUseDeveloperSupport() = BuildConfig.DEBUG
+
+        override fun getPackages(): List<ReactPackage> {
+            val packages = PackageList(this).getPackages()
```

Comment:
does `.getPackages()` return `List` or `MutableList`? Assuming the intention here was to allow manual additions here, it won't work in Kotlin because `List` doesn't have `.add` methods